woodwork/column_accessor.py

```diff
@@ -67,6 +68,49 @@ def description(self, description):
         _validate_description(description)
         self._schema['description'] = description
 
+    @property
+    def iloc(self):
+        """
+        Integer-location based indexing for selection by position.
+        ``.iloc[]`` is primarily integer position based (from ``0`` to
+        ``length-1`` of the axis), but may also be used with a boolean array.
```

Comment:
Would it make sense to have a line in here about how if the result is a Series it has Woodwork typing information still initialized?

Reply:
Sure, I can make that update.